classroom.rb

```diff
@@ -0,0 +1,15 @@
+require_relative './student'
+
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students << student
+  end
```

Comment:
- [ ] Remember that a `classroom` has many `students` and a `student` belongs to many `classrooms`. It would be nice to include a way for the student to set its classroom after pushing itself to the `students` array. Kindly add the line of code that does this after pushing a single student to the `students` array. You can read this [article]( https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) for a better understanding. 👍